projects/duplicate-char-counter/_posts/2020-10-04-python.md

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I don't believe we need this file. The index file below should cover everything here.